src/components/PeoplePage.tsx

```diff
@@ -1,31 +1,143 @@
+import { useEffect, useState } from 'react';
+import { useLocation, useSearchParams } from 'react-router-dom';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { getSearchWith } from '../utils/searchHelper';
+import { SortType } from '../types/SortType';
 
 export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
```

Comment:
Would be better to use an empty array instead
```suggestion
  const [people, setPeople] = useState<Person[]>([]);
```